posthog/temporal/batch_exports/s3_batch_export.py

```diff
@@ -550,10 +550,23 @@ async def insert_into_s3_activity(inputs: S3InsertInputs) -> RecordsCompleted:
         get_s3_key(inputs),
     )
 
+    start_at = dt.datetime.fromisoformat(inputs.data_interval_start) if inputs.data_interval_start is not None else None
+    end_at = dt.datetime.fromisoformat(inputs.data_interval_end)
+
+    if start_at:
+        is_5_min_batch_export = (end_at - start_at) == dt.timedelta(seconds=300)
+    else:
+        is_5_min_batch_export = False
+
+    if is_5_min_batch_export:
+        clickhouse_url = settings.CLICKHOUSE_OFFLINE_5MIN_CLUSTER_HOST
+    else:
+        clickhouse_url = None
+
     async with (
         Heartbeater() as heartbeater,
         set_status_to_running_task(run_id=inputs.run_id, logger=logger),
-        get_client(team_id=inputs.team_id) as client,
+        get_client(team_id=inputs.team_id, clickhouse_url=clickhouse_url) as client,
```

Comment:
I think that in a follow-up PR I'll move the clickhouse URL setting inside the `get_client` method itself. Since I want to limit the initial rollout to just S3, I am passing it here, but I will want to change this later.